components/builder-api/src/depot/server.rs

```diff
@@ -2654,6 +2827,94 @@ fn do_cache_response(response: &mut Response) {
     )));
 }
 
+fn set_memcache_package(
+    client: &mut memcache::Client,
+    origin: String,
+    name: String,
+    version: Option<String>,
+    channel: Option<String>,
+    key: &str,
```

Comment:
Is this actually the value?  From the memcache crate examples, it looks like the first arg is the key and the second is the value.  If that's the case, probably worth renaming this.